backend/src/question_user_response.ts

```diff
@@ -0,0 +1,65 @@
+import { APIGatewayEvent, ProxyResult } from 'aws-lambda';
```

Comment:
weird path names not very RESTful

Reply:
i agree, i was just copying the format Toan used. Not sure what to call it

Reply:
well fix it later
 im gonna merge this in for now